How to Find and Use Technical Resources if You’re New to Tech:proposal.md

```diff
@@ -0,0 +1,93 @@
+
+That's why we've compiled a list of tips on finding useful coding resources and some of our personal favorites. If you feel like you've hit a wall with your code, we hope you'll finding something that helps you out below! 
+
+## The Power of Googling
+
+Search engines are your best friends, and this is the number one way to find resources online because the internet is always there for you, no matter what time of day. 
+
+If you can't find what you're looking for right away, it can feel pretty frustrating. But it's important to not give up! 
+
+According to one of Bit Project's developers, Julia Ma, "One mistake I see a lot of  beginners make is that they don’t Google. When it comes to a problem that is coding-related — syntax, error, code compiling — always Google before you ask somebody because that’s how you learn the best."
+
+![](https://i.imgur.com/LyV2EDk.png)
+
+Sometimes you have to get specific with your search to get faster, better results. For example, you can get creative with what types of resources you look for: walkthroughs, documentation, [blog posts about coding](https://medium.com/topic/programming), etc. Or you can copy and paste the code you're having issues with directly into Google.
+
+No matter how you do it, Googling is a vital skill that will guide you towards countless of helpful coding resources. 
+
+## Stack Overflow 
+
+Every single Bit Project member I have talked to mentioned [Stack Overflow](https://stackoverflow.com/). They all cited this resource as one they always turn to in times of need.
+
+![](https://i.imgur.com/D1WrwSL.png)
+
+Stack Overflow provides you with a large, detailed library of questions that have been answered by professionals and other seasoned programmers. Everything you find on the site is meant for you to learn from, so don't hesitate to use it.
+
+"Stack Overflow is probably a software engineer's best friend," said Bryan Wong, director of engineering at Bit Project. "Any question you have, someone else is likely to have the exact same question as you. So if I’m really stuck on a problem, Stack Overflow is the way to go."
+
+## GeeksforGeeks 
+
+Another frequently mentioned resource was [GeeksforGeeks](https://www.geeksforgeeks.org/). This site, like Stack Overflow, also provides you with lots of detailed explanations on code. But while Stack Overflow is great for looking up answers for specific questions, GeeksforGeeks helps you strengthen your understanding of the concept behind the code.
+
+![](https://i.imgur.com/j6Ke9X1.png)
+
+Many members of Bit Project — like Atul Jayaram, QA manager — pointed to this site as a great resource if you're struggling to conceptualize programming languages. Specifically, they said GeeksforGeeks was a great place to quickly learn about C++ and Python, although there are lots of other tutorials offered by the organization that focus on a range of topics like algorithims, data structures, and numerous other langauges. 
```

Comment:
say "Atul Jayaram, **our** QA manager"
does Atul use they/them pronouns? it sounds like it here - if not, change the 'they' after 'Specifically' to 'he'
and is GeeksforGeeks an org? maybe say website?